webui/src/scss/_controls.scss

```diff
@@ -152,6 +167,19 @@ label.disabled {
 	}
 }
 
+/* Allow \n line-breaks and remove "excess" line spacing between pulldown options. */
+.select-control__option {
+	white-space: pre-wrap !important;
+	line-height: 1.1;
```

Comment:
These feel a bit squished together to me, but that could just be because I am used to how the variables one looks